src/data/navigation/sections/gateway.js

```diff
@@ -43,6 +43,10 @@ module.exports = [
                 title: 'Edge caching and cache control',
```

Comment:
I think we should rename edge caching term entirely to make space for Dynamic content caching, which could be done through CDN content delivery and API Mesh caching  (which we call server-side caching - IDK how to call that - if we do move API Mesh to the cloudflare it will become an edge caching - but not necessarily done through a CDN)

The feature we're describing as edge caching is simply being enabled by cache control headers which can be used by a CDN or client (browser) caching